pipelines/pipeline.yml

```diff
@@ -123,59 +114,16 @@ stages:
                     # Set the namespace as a Azure Pipeline variable
                     # See https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#set-in-script
                     echo "##vso[task.setvariable variable=namespace]${NS}"
+                    
+                    # We'll also write it to a file, so we can use it from other stage
+                    # (variables can be used across steps and jobs, but not across stages)
+                    echo "${NS}" > ./namespace.txt
+                    
                   displayName: "Configure AWS Namespace"
                   env:
                     PR_NUMBER: $(System.PullRequest.PullRequestNumber)
                     # This is the branch name, or the git tag name
                     NS_BRANCH_OR_TAG: $(Build.SourceBranchName)
-                # Install Terraform
-                - task: TerraformInstaller@0
-                  inputs:
-                    terraformVersion: "0.12.18"
-                  displayName: "Install Terraform"
-
-                - bash: |
-                    echo '
-                    terraform {
-                      backend "s3" {
-                        region = "us-east-1"
-                      }
-                    }
-                    ' > backend.tf
-                  displayName: "Generate backend.tf"
-                  workingDirectory: "$(Pipeline.Workspace)/modules"
-
-                # terraform init
-                - task: TerraformTaskV1@0
```

Comment:
We can't really use these TerraformTasks anymore, because they require static credentials. We want to use our dynamic creds from our DCE lease.

They didn't add much value anyways -- an unnecessary abstraction around a single CLI command, imo